.gitignore

```diff
@@ -1,4 +1,4 @@
-/node_modules/
+node_modules/
```

Comment:
both `packages/eslint-config-spt/node_modules` and `packages/eslint-config-spt-react/node_modules` exists. Top level have no `node_modules` except for the linked `eslint-config-spt`